tests/functional/codegen/modules/test_interface_imports.py

```diff
@@ -0,0 +1,31 @@
+def test_import_interface_types(make_input_bundle, get_contract):
+    ifaces = """
+interface IFoo:
+    def foo() -> uint256: nonpayable
+    """
+
+    foo_impl = """
+import ifaces
+
+implements: ifaces.IFoo
+
+@external
+def foo() -> uint256:
+    return block.number
+    """
+
+    contract = """
+import ifaces
+
+@external
+def test_foo(s: ifaces.IFoo) -> bool:
+    assert s.foo() == block.number
+    return True
+    """
+
+    input_bundle = make_input_bundle({"ifaces.vy": ifaces})
```

Comment:
```suggestion
    input_bundle = make_input_bundle({"ifaces.vyi": ifaces})
```

Reply:
going to skip this since i want to phase out `.vyi` files anyway